credsweeper/__main__.py

```diff
@@ -208,7 +218,9 @@ def scan(args: Namespace, content_provider: FilesProvider, json_filename: Option
                                   ml_threshold=args.ml_threshold,
                                   find_by_ext=args.find_by_ext,
                                   depth=args.depth,
-                                  size_limit=args.size_limit)
+                                  size_limit=args.size_limit,
+                                  exclude_lines=denylist,
+                                  exclude_values=denylist)
```

Comment:
deny_lines
deny_values
- maybe use the 'deny'' ? It helps to keep semantic in code.

Reply:
But we have exclude in config T-T
So it looks very nice when assigning to the config 

![image](https://user-images.githubusercontent.com/43581724/187880919-6b698945-0f26-40fd-b2db-9c1c7604fbe1.png)

Reply:
with single 'denylist' we will have double work. 
So, i propose use only single denylist in config

Reply:
> with single 'denylist' we will have double work. So, i propose use only single denylist in config

@babenek We are taking about O(1) average difficultly at the search operation
https://wiki.python.org/moin/TimeComplexity

Reply:
* N lines